projects/sdk-wells/src/lib/Well.ts

```diff
@@ -235,20 +239,23 @@ export class Well {
     amountIn: TokenValue,
     minAmountOut: TokenValue,
     recipient: string,
+    _deadline: number | string,
```

Comment:
Does the contract accept number | string?

Reply:
I should fix this, looking at it again it should just be a string.